scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/data/TaskProActiveDataspaces.java

```diff
@@ -98,23 +99,25 @@
 
     private SpaceInstanceInfo cacheSpaceInstanceInfo;
 
-    private transient StringBuffer clientLogs = new StringBuffer();
-
     private transient ExecutorService executorTransfer = Executors.newFixedThreadPool(getFileTransferThreadPoolSize(),
                                                                                       new NamedThreadFactory("FileTransferThreadPool"));
 
+    private transient TaskLogger taskLogger;
+
     /**
      * Mainly for testing purposes
      */
     TaskProActiveDataspaces() {
 
     }
 
-    public TaskProActiveDataspaces(TaskId taskId, NamingService namingService, boolean isRunAsUser) throws Exception {
+    public TaskProActiveDataspaces(TaskId taskId, NamingService namingService, boolean isRunAsUser,
```

Comment:
To limit the impact of this PR, keeping the old constructor along side w/ the new one requiring the TaskLogger could have been an option.

Reply:
ok, I will keep the previous constructor